getting-started/kubernetes/managed-public-cloud/eks.md

```diff
@@ -11,13 +11,53 @@ Enable Calico in EKS managed Kubernetes service.
 
 EKS has built-in support for Calico, providing a robust implementation of the full Kubernetes Network Policy API. EKS users wanting to go beyond Kubernetes network policy capabilities can make full use of the Calico Network Policy API.
 
+You can also use Calico for networking on EKS in place of the default AWS VPC networking. This allows you to take advantage of the full set of Calico networking features, including Calico's flexible IP address managemenet capabilities.
+
 ### How to
 
-To enable Calico network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Installing Calico on Amazon EKS' url='https://docs.aws.amazon.com/eks/latest/userguide/calico.html' %}
+#### Using VPC networking
 
 The geeky details of what you get:
 {% include geek-details.html details='Policy:Calico,IPAM:AWS,CNI:AWS,Overlay:No,Routing:VPC Native,Datastore:Kubernetes' %}
 
+To enable Calico network policy enforcement on an EKS cluster using the AWS VPC CNI plugin, follow these step-by-step instructions: {% include open-new-window.html text='Installing Calico on Amazon EKS' url='https://docs.aws.amazon.com/eks/latest/userguide/calico.html' %}
+
+#### Using Calico networking
+
+The geeky details of what you get:
+{% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes' %}
```

Comment:
You'll need to create a `_includes/geek-details/routing-calico.md` file to back this, since we don't yet have one.  You can look at the existing .md files in that directory to get a template for what to do.  It's important they don't have headings though, otherwise those show up in the right hand TOC, IIRC.